config/nav.yml

```diff
@@ -239,6 +239,8 @@ nav:
       - Client: reference/client/README.md
       - Concepts:
         - Duck types: reference/concepts/duck-typing.md
+    - About:
```

Comment:
We don't want a new menu "About"

I think this is what we are looking for
```
    - Community:
      - Welcome to the community: community/README.md
      - Contribute to Knative: community/contributing.md
      - About the community: community/about.md
    - About:
      - Testimonials: about/testimonials.md
      - Case studies:
        - about/case-studies/index.md
        - deepc: about/case-studies/deepc.md
        - Outfit7: about/case-studies/outfit7.md
        - Puppet: about/case-studies/puppet.md
    - Blog: /blog/
```

Reply:
One thing that is missing here is the index on the case studies, for now I'll redirect it to the deepc as it is right now

Reply:
yeah redirect to deepc one